config/config-es.json

```diff
@@ -60,7 +61,7 @@
         "focusBreakNotificationTitle": "Su descanso termina en",
         "tabGeneralText": "General",
         "tabFocusText": "Enfoque",
-        "tabSettingsText": "Configuraciones",
+        "tabEditHabitsText": "Configuraciones",
```

Comment:
Shouldn't this be Edit Habits?

Reply:
okay you mean the text in spanish...Right it should be Edit Habits.